src/ServicePulse.Host/vue/src/composables/isRouteSelected.ts

```diff
@@ -11,5 +11,6 @@ export default function isRouteSelected(path: string) {
     return true;
   }
 
-  return useLink({ to: path }).route.value.name === route.name;
+  const pathRoute = useLink({ to: path }).route.value;
+  return route.matched.some((match) => match.name === pathRoute.name);
```

Comment:
i actually don't think the two checks above are required anymore

Reply:
I agree, it seems the "matched" contains the routes matched in order of parent -> child -> grandchild -> ...